include/SFML/Audio/SoundRecorder.h

```diff
@@ -132,5 +134,58 @@ CSFML_AUDIO_API sfBool sfSoundRecorder_isAvailable(void);
 ////////////////////////////////////////////////////////////
 CSFML_AUDIO_API void sfSoundRecorder_setProcessingInterval(sfSoundRecorder* soundRecorder, sfTime interval);
 
+////////////////////////////////////////////////////////////
+/// \brief Get a list of the names of all availabe audio capture devices
+///
+/// This function returns an array of strings, containing
+/// the names of all availabe audio capture devices.
+///
+/// \param count Pointer to a variable that will be filled with the number of modes in the array
+///
+/// \return An array of strings containing the names
```

Comment:
You should add that in case of 0 device, it returns NULL (and not a valid pointer to an empty array).